app_dart/lib/src/datastore/cocoon_config.dart

```diff
@@ -398,6 +398,14 @@ class Config {
   bool githubPresubmitSupportedRepo(String repositoryName) {
     return supportedRepos.contains(repositoryName);
   }
+
+  Future<RepositorySlug> repoNameForBuilder(String builderName) async {
+    final List<Map<String, dynamic>> builders = luciTryBuilders;
+    final String repoName = builders.firstWhere(
```

Comment:
is it possible that different repos share same builder name?

Reply:
I think is possible although the current ones are all different. This method will get deprecated in a follow up PR where we'll move builder configs to the different repos.